scipy/signal/filter_design.py

```diff
@@ -2452,12 +2548,33 @@ def butter(N, Wn, btype='low', analog=False, output='ba'):
     >>> plt.axvline(100, color='green') # cutoff frequency
     >>> plt.show()
 
+    Generate a signal made up of 10 Hz and 20 Hz, sampled at 1 kHz
+
+    >>> t = np.linspace(0, 1, 1000)  # 1 second
```

Comment:
You need `endpoint=False` otherwise the effective sample rate is actually 999 Hz. This common mistake is one reason I usually opt for explicit division of sample numbers by `fs`, i.e. `np.arange(1000) / 1000.`